pkg/config/config_template.yaml

```diff
@@ -618,7 +618,7 @@ api_key:
     #
     # loader: core
 
-{{- if .Profiling -}}
+{{- if .InternalProfiling -}}
 ## @param profiling - custom object - optional
 ## Enter specific configurations for internal profiling.
 ## Uncomment this parameter and the one below to enable profiling.
```

Comment:
While you're at it, can you also update this comment to more explicitly state (1) that this does _not_ enable profiling for user applications, (2) that it only enables internal profiling of the agent runtime and (3) that users should refer to https://docs.datadoghq.com/tracing/profiling/ to enable profiling for their apps?